Backpack/Calendar/Classes/BPKCalendarPresenter.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@class BPKSimpleDate;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol BPKCalendarPresenting <NSObject>
+
+- (NSDate *)firstValidDayOfMonth:(NSDate *)month;
+- (NSDate *)lastDayOfMonth:(NSDate *)month;
+- (NSArray<BPKSimpleDate *> *)dateListForMonth:(NSDate *)month;
+
+@end
+
+@interface BPKCalendarPresenter : NSObject <BPKCalendarPresenting>
```

Comment:
This name is a little bit misleading, as it's not presenting anything. Can we change it to 'Formetter', 'DateUtils', 'DateProvider', 'DateResolver' or similar?